data/geolocation-!cn

```diff
@@ -71,6 +71,7 @@ include:mozilla
 include:nvidia
 include:oracle
 include:razer
+include:riot
```

Comment:
It's more suitable to include under `# Entertainment & Videos & Games & Music & Podcasts`.

Reply:
OK.

I'm in busy now, please mark this pr as draft.

Reply:
Only lpl.cn is for sale, right? You show L45-L48 in comment.

Reply:
> This domain is for sale.

Removed.

> It's more suitable to include under `# Entertainment & Videos & Games & Music & Podcasts`.

Fixed.

Reply:
> Only lpl.cn is for sale, right?

Yes. Thanks for your fix.

> You show L45-L48 in comment.

Did I? That's a mistake.